src/components/SingleAlbum.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function SingleAlbum({ album }) {
+  return (
+    <li>
+      <h3>{album.name}</h3>
+      <p>{album.artist}</p>
+      <img alt="album-cover-show" src={album.cover} />
```

Comment:
Por temas de accesibilidad, podríamos utilizar un `alt` más descriptivo, como por ejemplo:
```suggestion
      <img alt={`${album.name}, album from ${album.artist}`} src={album.cover} />
```